nimbus-ui/nimbusui/src/app/services/page.service.ts

```diff
@@ -61,6 +61,9 @@ export class PageService {
         gridValueUpdate = new Subject<Param>();
         gridValueUpdate$ = this.gridValueUpdate.asObservable();
 
+        treeListUpdate = new Subject();
```

Comment:
do we need a new subject for the tree table component? if there is nothing different that the observer and subscriber, can we reuse what we have?